bitnami/metallb/values.yaml

```diff
@@ -756,7 +756,7 @@ speaker:
   ## @param speaker.containerSecurityContext.runAsGroup Set containers' Security Context runAsGroup
   ## @param speaker.containerSecurityContext.allowPrivilegeEscalation Enables privilege Escalation context for the pod.
   ## @param speaker.containerSecurityContext.readOnlyRootFilesystem Allows the pod to mount the RootFS as ReadOnly
-  ## @param speaker.containerSecurityContext.capabilities.drop [array] Drop capabilities for the securityContext
+  ## @param speaker.containerSecurityContext.capabilities.drop [array] Drop capabilities for the securityContext. You need to comment out this block if you would like to use `tcpdump` for debugging purposes.
```

Comment:
Wouldn't it make more sense to add the comment right above line 772?

Reply:
I have just added it too.